sdk/daml-lf/transaction/src/main/scala/com/daml/lf/transaction/Transaction.scala

```diff
@@ -667,19 +667,16 @@ object Transaction {
 
   /** Transaction meta data
     *
-    * @param submissionSeed   : the submission seed used to derive the contract IDs.
-    *                         If undefined no seed has been used (the legacy contract ID scheme
-    *                         have been used) or it is unknown (output of partial reinterpretation).
-    * @param submissionTime   : the submission time
-    * @param usedPackages     : the set of all packages that are needed for the interpretation of the
-    *                         command (irrespective of whether the package needed to be loaded into
-    *                         cache via a [[com.daml.lf.engine.ResultNeedPackage]] callback.
-    * @param dependsOnTime    : indicate the transaction computation depends on ledger
-    *                         time.
-    * @param nodeSeeds        : An association list that maps to each ID of create and exercise
-    *                         nodes its seeds.
-    * @param globalKeyMapping : input key mapping inferred by interpretation
-    * @param disclosedEvents    : disclosed create events that have been used in this transaction
+    * @param submissionSeed   Populated with the submission seed when returned from [[com.daml.lf.engine.Engine.submit]].
+    * @param submissionTime   The submission time
+    * @param usedPackages     The set of all packages that are needed for the interpretation of the command. This
+    *                         will be all the [[Node.Action.packageIds]] used in the output transaction action nodes
```

Comment:
The term "used in" is too vague for my purposes. E.g., I suppose you do not consider pacakgeIds included in some contract data as "used".
Could you please add a pointer to a function (can be private) that computes the used packages, to avoid any ambiguity?